resources/common/plugins/template/cvvc-oto-gen/plugin.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "cvvc-oto-gen",
-  "version": 8,
+  "version": 9,
```

Comment:
Could you check that the parameters saved in the previous version can be properly loaded in the new version?
- Removed parameters should be ignored
- New parameters should be using default values

Reply:
Looks like everything happens as expected when using the new version. Unchanged parameters still keep their saved values and new parameters initalize with the default values.